.github/workflows/release-docs

```diff
@@ -0,0 +1,123 @@
+name: Release docs
+
+on:
+  push:
+    branches:
```

Comment:
Is there a way to avoid the duplicated workflow?

Reply:
The best I could come up with was skip wing release, and still trigger docsite update but that'd require a logic change on even the docsite repo. Or we could skip both for pure doc changes. The only solution that allows wing release and also skips tests which I found was this. Ofc I'm open to suggestions.